apps/etterlatte-tidshendelser/README.md

```diff
@@ -15,6 +15,11 @@ Denne appen styrer jobber som kjører med jevne mellomrom for å håndtere netto
     - NB: Mangler sjekk for utvidelse med 2 år (sannsynligvis nytt vilkår) 
   - `OMS_DOED_5AAR` Opphør utvidet omstillingsstønad, (3 + 2) år etter dødsfall
     - Unntak: har rett uten tidsbegrensning  
+- Regulering
+  - For å trigge denne jobben gjøres følgende:
+  1. Gå inn i Unleash og aktiver flagget start-regulering
+  2. Restart applikasjonen, for eksempel ved å slette podden (kubectl delete pod) eller redeploye 
+  3. Etter endt kjøring, skru av reguleringsflagget igjen
```

Comment:
En ting jeg slet litt med å forstå er når de ulike jobbene kjører. Kunne det har blitt dokumentert mer her?

Jeg tolket det sånn at FINN_SAKER_TIL_REGULERING og REGULERING kjøres hver for seg? Slik at vi først gjør klar alle sakene og deretter starter reguleringsjobben med et gitt antall eller spesifikt angitte?
Jeg har ikke satt meg så veldig inn i tidshendelserløsningen så hvordan jobbene faktisk iverksettes forsto jeg ikke helt. Sendes det hendelse 1. mai? Kontinuerlig (derav feature toggle)?

Reply:
For aldersovergangsjobbene i `tidshendelser` så står det en egen lytter og poller på en jobb-tabell, hvor det er angitt _type_ jobb, og _hvilken dato den skal kjøres_. Jobben kjører, og inserter så rader i en hendelsetabell (en for hver sak den identifiserer som aktuell) - som så en annen mekanisme poller kontinuerlig, med en gitt limit for hvor mange rader som behandles pr poll.

Reply:
Aha ja - med andre for for Regulering så vil vi inserte og delete før og etter når skal regulere?

Reply:
Jeg tror Mads må svare her. Jeg har bladd med gjennom PRen lokalt, commit for commit, og jeg er ikke helt sikker på målbildet :)

Reply:
> En ting jeg slet litt med å forstå er når de ulike jobbene kjører. Kunne det har blitt dokumentert mer her?
> 
> Jeg tolket det sånn at FINN_SAKER_TIL_REGULERING og REGULERING kjøres hver for seg? Slik at vi først gjør klar alle sakene og deretter starter reguleringsjobben med et gitt antall eller spesifikt angitte? Jeg har ikke satt meg så veldig inn i tidshendelserløsningen så hvordan jobbene faktisk iverksettes forsto jeg ikke helt. Sendes det hendelse 1. mai? Kontinuerlig (derav feature toggle)?

Ja, bør kanskje dokumentere meir her.

Du har tolka det rett (eller i det minste likt som eg har tolka det). Først gjer vi `FINN_SAKER_TIL_REGULERING`, og så rett etterpå kan vi starte med `REGULERING`. Har sett for meg at vi køyrer inserts i databasen tilsvarande som vi gjorde for migrering.

Denne modellen bruker berre jobb-biten, og unngår å pushe inn i hendelse, så det er ein litt annan modell enn resten av applikasjonen her. Er usikker på om det er det endelege målbildet, eller om vi bør gjera noko meir.